background.js

```diff
@@ -2,6 +2,17 @@
 
 import { addToArchiveBox } from "./utils.js";
 
+class Entry {
+  constructor(url, tags, title, favIconUrl) {
+    this.id = crypto.randomUUID(),
```

Comment:
```suggestion
    this.id = crypto.randomUUID()
```

remove the commas after every line